feature-policy/header-policy-payment-disabled.sub.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+  if (window.testRunner) {
+    testRunner.dumpAsText();
+    testRunner.waitUntilDone();
+  }
+</script>
+<iframe id="same-origin" src="resources/vibrate-disabled.html"></iframe>
+<iframe id="cross-origin" src="{{location[scheme]}}://{{domains[www]}}:{{location[port]}}/feature-policy/resources/vibrate-disabled.html"></iframe>
+<iframe id="same-origin" src="resources/vibrate-disabled.html"></iframe>
```

Comment:
These two frames should have different ids from the ones in line 10 and 11 -- It's an error to have two elements in the same document with the same id attribute
(https://html.spec.whatwg.org/#the-id-attribute)